src/widgets/Paper.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View } from 'react-native';
+
+import { WHITE } from '../globalStyles';
+
+export const Paper = ({ width, height, style, children }) => {
+  // If width and height are passed hard fix them in the container style,
+  // let the axis which isn't passed flex. Plain width or height properties
+  // sometimes act as suggestions when the container is flexing so use
+  // max/min properties.
+  const internalContainerStyle = { ...style };
```

Comment:
an alternative is possibly to disallow flex grow & shrink and to set the initial size:

`flex: '0 ${width}px'`

which retains flex, but then doesn't help with height so is perhaps not useful